agents/postgres/pgstatmonitor/pgstatmonitor.go

```diff
@@ -247,10 +248,11 @@ func (m *PGStatMonitorQAN) makeBuckets(current, cache map[time.Time]map[string]*
 				m.l.Debugf("Normal query: %s.", currentPSM)
 			}
 
+			fingerprint, _ := truncate.Query(currentPSM.Fingerprint)
```

Comment:
We already truncate fingerprint in https://github.com/percona/pmm-agent/blob/PMM-5283-truncate-query-example-and-fingerprint/agents/postgres/pgstatmonitor/stat_monitor_cache.go#L169

Reply:
Fixed.